shared/src/iosMain/kotlin/com/innova/innovaPoolingLibrary/DateFormatter.kt

```diff
@@ -0,0 +1,17 @@
+package com.innova.innovaPoolingLibrary
+
+interface DateProtocol {
+    fun getIosDate(): String
+}
+
+actual class DateFormatter {
+    private lateinit var dateProtocol: DateProtocol
+
+    fun setImplementation(protocol: DateProtocol) {
+        dateProtocol = protocol
```

Comment:
Is there any problem in setting this in initialiser?

Reply:
Do you want me to initialise the dateProtocol variable in the constructor of class DateFormatter?